homeassistant/components/rflink/__init__.py

```diff
@@ -323,7 +323,6 @@ class RflinkDevice(Entity):
     Contains the common logic for Rflink entities.
     """
 
-    platform = None
```

Comment:
This looks like a leftover and safe to remove. Did you come to the same conclusion?

Reply:
Yes, the only thing this did was to remove the type annotation which would otherwise have been inherited from `Entity`.